windows.versions

```diff
@@ -1,5 +1,17 @@
 {
   "windows": [
+    {
+      "version": "2.31.8",
+      "build": "1",
+      "fluent-bit": "1.9.10",
+      "kinesis-plugin": "v1.10.1",
+      "firehose-plugin": "v1.7.1",
+      "cloudwatch-plugin": "v1.9.2",
```

Comment:
The go plugins should be updated as well, remember I noted on Monday we need to create a new tag in reach repo for dependency upgrades.